test/unittests/execute_call_test.cpp

```diff
@@ -531,55 +531,6 @@ TEST(execute_call, imported_function_from_another_module_via_func_idx)
     EXPECT_THAT(execute(*instance2, 0, {44, 2}), Result(42));
 }
 
-TEST(execute_call, imported_function_from_another_module_via_host_function)
```

Comment:
I think this test might be ok to leave, it was not about call depth. It just demonstrates technique of passing another instance + function index into host function.

Optionally host function can be changed to bump the depth, I think it's not very important here.

Reply:
Restored.